rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/interpretation_utils.kt

```diff
@@ -65,3 +65,20 @@ fun ActivationGroupType.assignedName(current: RpgProgram, caller: RpgProgram?):
         else -> error("$this ActivationGroupType is not yet handled")
     }
 }
+
+fun DataDefinition.resizeStringSize(newSize: Int) {
```

Comment:
If we don't have to expose Jariko features, it is preferable to use the `internal` modifier, in addition I would like a little bit of documentation for both public and internal functions too, mostly for helper and utilty functions.
Take in account of this for the future.

Reply:
I added a bit of documentation. Tell me if is ok.

Reply:
You should explain the logic, for instance: the method transforms the type from varying to not varying and if the target size is shorter the content will be truncated...

Reply:
Done